tests/rkt_net_test.go

```diff
@@ -200,6 +200,68 @@ func TestNetDefaultNetNS(t *testing.T) {
 	f("")
 }
 
+func TestNetGWv4Connectivity(t *testing.T) {
+	ctx := testutils.NewRktRunCtx()
+	defer ctx.Cleanup()
+
+	f := func(argument string) {
+		httpPort, err := testutils.GetNextFreePort4()
+		if err != nil {
+			t.Fatalf("%v", err)
+		}
+		httpServeAddr := fmt.Sprintf("0.0.0.0:%v", httpPort)
+		httpServeTimeout := 30
+		iface := "eth0"
```

Comment:
@alban update interface by calling  ``` testutils.GetNonLoIfaceIPv4 ```, and removed the duplicate interface designating. Please check, thanks!